releasenotes/notes/depends-on-six-e9b79edbf0d169a2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The `six` library has been removed from `vendor` and the system-wide version is being used.
```

Comment:
Might be worth mentioning the minimum version required as that could be a breaking change is someone has a really old version of six pinned.